garglk/fontwin.c

```diff
@@ -228,6 +191,62 @@ int CALLBACK propfont(
     return 1;
 }
 
+static void make_font_filepath(const char *filename, char *filepath, size_t length)
+{
+    // create the absolute path to the font file
+    filepath[0] = '\0';
+    if (!strstr(filename, ":") && getenv("SYSTEMROOT"))
```

Comment:
As long as this is being touched, it really ought to be updated to be safe against buffer overflows.  There are lots of places in Gargoyle that are (potentially) susceptible to overflows, and they should all be addressed at some point, but at least we can plug some of them now.  I'd recommend something like:

```c
if (!strstr(filename, ":") && getenv("SYSTEMROOT"))
    snprintf(filepath, length, "%s\\Fonts\\%s", getenv("SYSTEMROOT"), filename);
else
    snprintf(filepath, length, "%s", filename);
```